tcms/testcases/templates/testcases/search.html

```diff
@@ -43,7 +43,7 @@
                 <select class="form-control selectpicker" id="id_product">
                     <option value="">----------</option>
                 {% for option in form.product.field.queryset %}
-                    <option value="{{ option.pk }}">{{ option.name }}</option>
+                    <option value="{{ option.pk }}" {% if form.product.value|add:'0' == option.pk %}selected{% endif %}>{{ option.name }}</option>
```

Comment:
Good catch.

In some templates we also use `product.pk|escape == form.product.value|escape` in the same construct. Appears to be a bit more frequently used. If you are up for making everything unified then grep for "|add:" and update!